apps/courses/factories.py

```diff
@@ -1,13 +1,48 @@
 """
 Courses factories
 """
-import factory
+from django.utils.text import slugify
 
 from cms.api import create_page
+import factory
+
+from .models import Course, Organization, Subject
```

Comment:
Surprising importation ordering. I think it's time to integrate [`isort`](https://github.com/timothycrosley/isort). If you agree, I can declare a new issue to integrate it later. WDYT?

Reply:
Why surprising this one? C < O < S ?

Reply:
No i'm targeting import statements order : from/import/from. `isort` would be useful here.

Reply:
Oh ok. I didn't know this rule and was applying alphabetical ordering.
Definitely +1 for isort.